lib/util/PDF/shareAndDownload.dart

```diff
@@ -537,15 +270,58 @@ Future<void> downloadPDF(List<dynamic> titles, List<dynamic> subTitles,
       header: (context) {
         return pw.Row(
           children: [
-            pw.SizedBox(height: 100, width: 100, child: Image),
+            pw.SizedBox(height: 100, width: 100, child: image),
           ],
         );
       },
       pageFormat: PdfPageFormat.a4,
       build: (context) => widgets, // Build the PDF with the generated widgets
     ),
   );
+  return pdf;
+}
+
+// Function to combine phone names and numbers into a formatted list
+List<String> setPhones(names, numbers) {
```

Comment:
This method is not clear. the name suggest it will set phones somewhere but it actually return a temp list.
Maybe it can be 'mergePhoneNumberAndName' ?

Reply:
Changed this, please see if the names are more clear now.